app/src/main/java/ar/com/wolox/android/example/model/User.java

```diff
@@ -0,0 +1,122 @@
+package ar.com.wolox.android.example.model;
+
+import com.google.gson.annotations.Expose;
+import com.google.gson.annotations.SerializedName;
+
+/**
+ *
+ */
+public class User {
+
+    @SerializedName("id")
```

Comment:
Oh my eyes :roll_eyes: Try to write attributes on a single line. For example: `@SerializedName("id") @Expose private Integer id;`

Reply:
Hey! this is still the same

Reply:
Sorry, bad news, checkstyle didn't let me to simplify in a single line, so I had to take two :(

Reply:
Ok, I'm starting to hate checkstyle :smile: